domain.yml

```diff
@@ -1,753 +1,862 @@
-  utter_ask_voice:
-    - text:  "Yes, that is possible! You can connect Rasa to any channel you like. You can find more information [here](https://rasa.com/docs/core/connectors/)."
-  utter_faq_platform_more:
-    - text: "Please read our [documentation](https://rasa.com/docs/platform/) for more information."
-  utter_faq_language_more:
-    - text: "Please read our [documentation](https://rasa.com/docs/nlu/languages/) for more information."
-  utter_faq_voice_more:
-    - text: "Please read our [documentation](https://rasa.com/docs/core/connectors/) for more information."
-  utter_faq_slots_more:
-    - text: "Please read our [documentation](https://rasa.com/docs/core/slots/) for more information."
-  utter_faq_channels_more:
-    - text: "Please read our [documentation](https://rasa.com/docs/core/connectors/#creating-a-new-channel) for more information."
+  - text: Now, let's take a look at Rasa X - the tool that helps Rasa learn from real
+      conversations.
+  utter_get_started_step4:
+  - text: As Rasa developer you will be part of a huge community. You can ask me which
+      events we have planned, how to contribute, or about our Rasa forum.
+  utter_getstarted:
+  - text: To determine how I can help you best, I'm going to ask you a few questions.
   utter_getstarted_new:
-    - text: "I see you're new, let me ask you a quick question before I help you get started:"
-  utter_ask_feedback_message:
-    - text: "I’m sorry I couldn't help you. Please let me know if you have any suggestions for how I can improve."
-  utter_thanks_for_feedback:
-    - text: "Thanks! Check out our [forum](https://forum.rasa.com), some of the smart people from our community can answer your questions there."
-  utter_restart_with_button:
-    - text: "Click the button below if you want to start over."
-      buttons:
-      - title: "Restart"
-        payload: "/restart"
+  - text: 'I see you''re new, let me ask you a quick question before I help you get
+      started:'
+  utter_great:
+  - text: "Great! 👍"
+  - text: "That's great! 🙌"
+  - text: "Awesome! 🎉"
+  - text: "Cool, let's do it 🚀"
+  utter_greet:
+  - text: Hey there, welcome to Rasa!
+  utter_greet_name:
+  - text: Hi, {name}!
+  - text: Hello, {name}!
+  - text: Hey, {name}!
+  utter_greet_noname:
+  - text: Hey!
+  - text: Hi!
+  - text: Hey there!
+  utter_handleinsult:
+  - text: "That's not very nice 😢"
+  utter_have_you_used_rasa_before:
+  - text: Have you used Rasa before?
+  utter_inform_privacypolicy:
+  - text: 'By chatting to me you agree to our privacy policy, which you can read here:
+      https://rasa.com/privacy-policy/.'
+  utter_installation_command:
+  - text: You will first need to install Rasa using this installation command
+      "pip3 install rasa-x --extra-index-url https://pypi.rasa.com/simple".
```

Comment:
This reflects the installation command on the installation page. @imLew made a suggestion that we have it as `pip3` so that we have a single command for both virtualenv and non-virtualenv users.

Reply:
Afaik `pip3` will always work if you have python3 with pip installed, so this should make it just a bit easier users who are new to python who might not have their environments properly configured.

Reply:
if i use pip3 in my virtualenv it does not work for me because my default is python3